instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import Comment from './Comment';
+import PropTypes from 'prop-types';
+import './comment-section.css';
+
+
+class CommentSection extends Component {
+    constructor (props) {
+        super(props);
+        this.state = {
+            comments: props.comments,
+            text: ''
+        };
+    }
+
+    changeHandler = (e) => {
+        this.setState({text: e.target.value})
+    }
+
+    addNewComment = (e) => {
+      e.preventDefault();
+      this.setState(prevState => {
+          return {
+              comments: [...prevState.comments,
+              {
+                 text: prevState.text,
+                 username: localStorage.getItem('username')
+              }],
+              text: ''
+          }
+      })
+    }
+
+    render() {
+      return (
+        <div>
+          {this.state.comments.map(comment => {
+            return <Comment comment={comment} key={comment.text} />;
```

Comment:
It may be helpful to use an index for the key value or a *uuid* because if I type the same text twice, both comments will have the same key.